bindata/network/ovn-kubernetes/managed/ovnkube-master.yaml

```diff
@@ -170,7 +170,7 @@ spec:
             -c /ovn-cert/tls.crt \
             -C /ovn-ca/ca-bundle.crt &
 
-          wait $!
```

Comment:
`sleep infinty`   may be easier than doing a loop

Reply:
I can do that.

Reply:
So what happens if the ovn thing (northd, controller, DBs, whatever) exits for some reason? Won't the sleep loop cover that up and leave the container running, while the process we care about is dead?

The `wait $!` would at least exit the container and let kubelet restart it, right? how does that work here?

Reply:
yes, that's correct. I can see that could be a major problem then.

hmmm....

Do we have to stop the process in the preStop hook? we have this code:

```
          quit() {
            echo "$(date -Iseconds) - stopping sbdb"
            /usr/share/ovn/scripts/ovn-ctl stop_sb_ovsdb
            echo "$(date -Iseconds) - sbdb stopped"
            rm -f /var/run/ovn/ovnsb_db.pid
            exit 0
          }
          # end of quit
          trap quit TERM INT
```
          
that is called on a SIGTERM which is what we get in normal cases with cri-o stopping the containers. Is that
good enough and we can just remove the prestop hooks?

btw, the 'echo' "rm -f" and "exit 0" lines will not exectute since the "ovn-ctl stop..." command kills things all
by itself.

Reply:
btw, I have [this PR](https://github.com/openshift/cluster-network-operator/pull/1965/commits/08951fde319539e8d40d540d6cd7097df7ee355b) removing the ovn-ctl stop command from the prestop hook. I know it gets us around the
issue I'm concerned with. what would be the downside of this?